examples/gno.land/p/demo/forms/create.gno

```diff
@@ -0,0 +1,121 @@
+package forms
+
+import (
+	"std"
+	"time"
+
+	"gno.land/p/demo/json"
+)
+
+func CreateField(label string, fieldType string, required bool) Field {
+	return Field{
+		Label:     label,
+		FieldType: fieldType,
+		Required:  required,
+	}
+}
+
+type times struct {
+	openAt  *time.Time
+	closeAt *time.Time
+}
+
+func parseDates(openAt string, closeAt string) (times, error) {
+	var openAtTime, closeAtTime *time.Time
+
+	const dateFormat = "2006-01-02T15:04:05Z"
+
+	// Parse openAt if it's not empty
+	if openAt != "" {
+		res, err := time.Parse(dateFormat, openAt)
+		if err != nil {
+			return times{}, errInvalidDate
+		}
+		openAtTime = &res
+	}
+
+	// Parse closeAt if it's not empty
+	if closeAt != "" {
+		res, err := time.Parse(dateFormat, closeAt)
+		if err != nil {
+			return times{}, errInvalidDate
+		}
+		closeAtTime = &res
+	}
+
+	return times{openAtTime, closeAtTime}, nil
+}
```

Comment:
1. this(`times`) seems to be a redundant design;
2. using time.Time directly (without a pointer) is perfectly fine.